README.md

```diff
@@ -87,7 +64,7 @@ To debug using CLion, you can create a run configuration using the Embedded Gdb
 4. For download executable, select ``None``. This will be done manually while launching the gdb server (aka openocd)
 5. For target remote args, enter ``localhost 3333``
 6. For the GDB server, choose the one provided with the esp tool chain, similarly to the gdb (example path: ``/home/casto/.espressif/tools/openocd-esp32/v0.10.0-esp32-20191114/openocd-esp32/bin/openocd``)
-7. For the GDB server args, enter these args, modifying the paths to your project: `` -s share/openocd/scripts -f /home/casto/git/Sherbrooke/SwarmUs/esp32-boilerplate/tools/openocd/adafruit-esp.cfg -c "program_esp /home/casto/git/Sherbrooke/SwarmUs/esp32-boilerplate/cmake-build-target/esp32-boilerplate.bin 0x10000"``
+7. For the GDB server args, enter these args, modifying the paths to your project: `` -s share/openocd/scripts -f /home/casto/git/Sherbrooke/SwarmUs/HiveConnect/tools/openocd/adafruit-esp.cfg -c "program_esp /home/casto/git/Sherbrooke/SwarmUs/HiveConnect/cmake-build-target/hive_connect.bin 0x10000"``
```

Comment:
Maybe provide a more "generic" example path than the one from your computer?

Reply:
Done